### DISCUSSION TITLE
Not starting

### DISCUSSION THREAD

**Discussion by EmiShoker** [phcode-dev/phoenix#1884] [Q&A]
cannot run phoenix code, 'no specific error message available', tryed uninstall, didn't work, help please?
![изображение_2024-10-14_000520379](https://github.com/user-attachments/assets/e74d8583-19b0-441c-b076-7332762d8932)

**Comment by abose** [comment]
What is the os version. Is there anything special about your system? For eg. Installed on a Network drive?

**Reply by EmiShoker** [reply]
Win 11 23h2, installed on a single main drive, when I installed it for the first time, everything was fine, a week later there was such an error
